src/index.html

```diff
@@ -9,10 +9,53 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="conteiner">
```

Comment:
There is a typo in the class name 'conteiner'. It should be 'container' to match common naming conventions and likely the CSS styles in 'style.css'.